code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -230,7 +232,7 @@ \subsection{Purpose of Document}
 This document will be used as a starting point for subsequent development phases, including writing the design specification and the software verification and validation plan. The design document will show how the requirements are to be realized, including decisions on the numerical algorithms and programming environment. The verification and validation plan will show the steps that will be used to increase confidence in the software documentation and the implementation. Although the SRS fits in a series of documents that follow the so-called waterfall model, the actual development process is not constrained in any way. Even when the waterfall model is not followed, as Parnas and Clements point out \cite{parnasClements1986}, the most logical way to present the documentation is still to ``fake'' a rational design process.
 \subsection{Scope of Requirements}
 \label{Sec:ReqsScope}
-The scope of the requirements includes stability analysis of a 2 dimensional slope, composed of homogeneous soil layers. Given the appropriate inputs, SSA identifies the most likely failure surface within the possible input range, and finds the factor of safety for the slope.
+The scope of the requirements includes stability analysis of a Two-Dimensional (2D) soil mass, composed of a single homogeneous layer with constant material properties. The soil mass is assumed to extend infinitely in the third dimension. The analysis will be at an instant in time; factors that may change the soil properties over time will not be considered.
```

Comment:
Hmm, the capitalization here isn't good on "Two-Dimensional". Or at least, not what I expected.

Reply:
Would "2-Dimensional" be better?

Reply:
In inline text, I expect "two-dimensional".  It's not a title of something.

Reply:
The reason for the capitalization is because this is where the "2D" abbreviation gets introduced. I thought (wrongly?) we should always capitalize for introduction of an acronym.

Reply:
I don't think so? Yes, it happens that most acronyms seem to work that way, but not abbreviations.